pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -361,9 +363,7 @@ def completed(self):
         if not keyboard.can_configure_keyboard():
             return True
 
-        # Request user attention if the current activated layout is a different from the
-        # selected ones
-        return self._xkl_wrapper.get_current_layout() in self._l12_module.XLayouts
+        return True
```

Comment:
Damn, I removed it because it was crashing and I forgot to re-enable it before opening the PR :man_facepalming: 

Now it is handled correctly. Please note that GNOME Kiosk sets it properties asynchronously, therefore, we can't fetch the layout here. Instead, we need to set it on the `compositor_selected_layout_changed` signal handler and compare it.